sopel/modules/admin.py

```diff
@@ -87,6 +98,7 @@ def _join(bot, channel, key=None, save=True):
         if channel not in channels or channels[channel] != key:
             channels[channel] = key
             _set_config_channels(bot, channels)
+            LOGGER.info('Updated core.channels with "%s".', channel)
```

Comment:
Parallel messages ❤️

```suggestion
            LOGGER.info('Added "%s" to core.channels.', channel)
```